// web-error/dist/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.RequestTimeout = exports.ProxyAuthenticationRequired = exports.NotAcceptable = exports.MethodNotAllowed = exports.NotFound = exports.Forbidden = exports.PaymentRequired = exports.Unauthorized = exports.BadRequest = exports.StatusCode = undefined;

var _WebError = require('./WebError');

var _WebError2 = _interopRequireDefault(_WebError);

var _StatusCode2 = require('./StatusCode');

var _StatusCode3 = _interopRequireDefault(_StatusCode2);

var _BadRequest2 = require('./BadRequest');

var _BadRequest3 = _interopRequireDefault(_BadRequest2);

var _Unauthorized2 = require('./Unauthorized');

var _Unauthorized3 = _interopRequireDefault(_Unauthorized2);

var _PaymentRequired2 = require('./PaymentRequired');

var _PaymentRequired3 = _interopRequireDefault(_PaymentRequired2);

var _Forbidden2 = require('./Forbidden');

var _Forbidden3 = _interopRequireDefault(_Forbidden2);

var _NotFound2 = require('./NotFound');

var _NotFound3 = _interopRequireDefault(_NotFound2);

var _MethodNotAllowed2 = require('./MethodNotAllowed');

var _MethodNotAllowed3 = _interopRequireDefault(_MethodNotAllowed2);

var _NotAcceptable2 = require('./NotAcceptable');

var _NotAcceptable3 = _interopRequireDefault(_NotAcceptable2);

var _ProxyAuthenticationRequired2 = require('./ProxyAuthenticationRequired');

var _ProxyAuthenticationRequired3 = _interopRequireDefault(_ProxyAuthenticationRequired2);

var _RequestTimeout2 = require('./RequestTimeout');

var _RequestTimeout3 = _interopRequireDefault(_RequestTimeout2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.StatusCode = _StatusCode3.default;
exports.BadRequest = _BadRequest3.default;
exports.Unauthorized = _Unauthorized3.default;
exports.PaymentRequired = _PaymentRequired3.default;
exports.Forbidden = _Forbidden3.default;
exports.NotFound = _NotFound3.default;
exports.MethodNotAllowed = _MethodNotAllowed3.default;
exports.NotAcceptable = _NotAcceptable3.default;
exports.ProxyAuthenticationRequired = _ProxyAuthenticationRequired3.default;
exports.RequestTimeout = _RequestTimeout3.default;
exports.default = _WebError2.default;
